hatch.toml

```diff
@@ -108,7 +108,8 @@ test = [
 extra-dependencies = [
     "pytest",
     "pytest-cov",
-    "hypothesis<6.45.1"
+    "hypothesis<6.45.1",
+    "mock"
```

Comment:
I think you can avoid this dependency by using `unittest.mock` from the Python standard library. If that's right, I'd recommend doing so.